src/briefcase/deployers/util-contracts/FeeCollectorDeployer.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >= 0.7.0;
+
+import {IFeeCollector} from '../../protocols/util-contracts/interfaces/IFeeCollector.sol';
+
+library FeeCollectorDeployer {
+    function deploy(address owner, address universalRouter, address permit2, address feeToken)
+        internal
+        returns (IFeeCollector feeCollector)
+    {
+        bytes memory args = abi.encode(owner, universalRouter, permit2, feeToken);
+        bytes memory initcode_ = abi.encodePacked(initcode(), args);
+
+        assembly {
+            feeCollector := create(0, add(initcode_, 32), mload(initcode_))
```

Comment:
do we want to use create2? or does it not matter?

Reply:
l want to do a separate PR with create2 for everything

Reply:
okay cool